src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/custom.py

```diff
@@ -419,13 +419,22 @@ def list_app_service_plans(resource_group_name=None):
     return plans
 
 
+def _linux_sku_check(sku):
+    tier = _get_sku_name(sku)
+    if tier == 'BASIC' or tier == 'STANDARD':
```

Comment:
what about all the other cases such as Premium (P1, P2, P3), Dynamic(D1, D), etc..